FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Deniz Tutuncu on 8/14/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+		var feed: [FeedImage] {
+			return items.map { $0.item }
+		}
+	}
+
+	private struct Item: Decodable {
+		let id: UUID
+		let description: String?
+		let location: String?
+		let imageURL: URL
+
+		private enum CodingKeys: String, CodingKey {
+			case id = "image_id"
+			case description = "image_desc"
+			case location = "image_loc"
+			case imageURL = "image_url"
+		}
+
+		var item: FeedImage {
+			return FeedImage(id: id, description: description, location: location, url: imageURL)
+		}
+	}
+
+	private static var OK_200: Int { return 200 }
+
+	internal static func map(_ data: Data, from response: HTTPURLResponse) -> RemoteFeedLoader.Result {
```

Comment:
`internal` is the default access level, so you don't need to write it explicitly. 

We write it explicitly in the first lectures for teaching purposes, but you don't need to do it in your code.

You can remove it.

```suggestion
	static func map(_ data: Data, from response: HTTPURLResponse) -> RemoteFeedLoader.Result {
```